### DISCUSSION TITLE
Arc entities are created clockwise or counterclockwise?

### DISCUSSION THREAD

**Discussion by lucasgamaleri** [haplokuon/netDxf#326] [Q&A] [answered]
Hey everyone, I have a question.

When I get startAngle and endAngle from an Arc entity. How should I visualize the entity? from startAngle to endAngle clockwise or counterclockwise?

This question is relevant to me because I want to calculate the top left and top right limits coordinates of an object made with lines and arcs.

Thanks in advance.

**Comment by haplokuon** [comment, marked as answer]
[Here](https://github.com/haplokuon/netDxf/issues/233#issuecomment-736249787).
